src/components/ActionsBar/index.tsx

```diff
@@ -0,0 +1,40 @@
+// Styles
+import './ActionsBar.scss'
+
+// Types
+type Props = {
+  filterBySub: Function
```

Comment:
Avoid 'any' type functions.

```suggestion
  filterBySub: (str: string) => void
```